### ISSUE TITLE
Funny teleport bug

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#13266]
(WEB REPORT BY: dorkson0 REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)
85cd902be3652fcd4b3785495c5d93f34d634657 

>General description of the issue
blue space bananas that teleport mobs on a tile that they are thrown at act funny if the mob is buckled in

>What you expected to happen
threw it at the floor while I was strapped in and expected it to teleport me

>What actually happened
threw the banana at myself while strapped into shuttle chair, which teleported me to somewhere else on the station, but I was still buckled into an invisible chair that I was unable to unbuckle from, until finally the shuttle left and I was teleported back to my chair in the shuttle automatically

>Steps to reproduce if possible

**Comment by Ephx1** [comment]
I remember experiencing the same thing and seeing others go through it. It's pretty old.

**Comment by 9600bauds** [comment]
I think this is also a thing if an alien buckles into a chair and ventcrawls while buckled

**Comment by PJB3005** [comment]
TL;DR botany bluespace warping effect ignores atom locking.

**Comment by ghost** [comment]
It just uses forcemove, doesn't actually unbuckle them

**Comment by Shadowmech88** [comment]
Fixed by #20134.
